### PR TITLE
update the setup instructions

### LINKED ISSUE #196: 0.1.0: Getting "Must provide query string." Response From Server
Given this GraphQL document:
```
query getAllTrips {
  allTrips {
    id
    title
    startTime
    duration
    creationTime
  }
}
```

...and this Retrofit interface:

```java
interface ServerApi {
  @POST("graphql")
  Call<Response<GetAllTrips.Data>> _allTrips(@Body GetAllTrips query);
}
```

...and this Java code to execute the query in the document:

```java
        Retrofit retrofit=new Retrofit.Builder()
          .baseUrl("https://graphql-demo.commonsware.com")
          .addConverterFactory(new ApolloConverterFactory.Builder().build())
          .addConverterFactory(MoshiConverterFactory.create())
          .build();
        ServerApi api=retrofit.create(ServerApi.class);

        try {
          retrofit2.Response<Response<GetAllTrips.Data>> response=
            api._allTrips(new GetAllTrips()).execute();
          Log.e("20170214", response.errorBody().string());
        }
        catch (IOException e) {
          Log.e("20170214", "Exception making query", e);
        }
```

...I get a 400 response from my server, and the error body of the response is:

```json
{"errors":[{"message":"Must provide query string."}]}
```

This feels like a bug, insofar as providing the query string is what the generated code is supposed to be doing, if I understand how this is all supposed to work.

The attached ZIP archive contains a complete sample project for reproducing this.

If I screwed up somewhere, I apologize.
[StaticList.zip](https://github.com/apollographql/apollo-android/files/775140/StaticList.zip)

### PR DESCRIPTION
**Pull request by digitalbuddha**
The readme had laughable setup instructions. Laughable as in how funny it was that the gradle instructions were incorrect/incomplete :-)
This pr adds proper build.gradle configurations

close #196

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,18 +9,83 @@ Apollo-Android is a GraphQL compliant client that generates Java models from sta
 
 
 ## Usage
+Add the following to your project's top level `build.gradle` file
 
 ```groovy
+
+ext {
+    kotlinVersion = '1.0.6'
```
Comment (sav007) [inline README.md]: Do we need this for sample?

File: README.md [hunk 2]
```diff
@@ -9,18 +9,83 @@ Apollo-Android is a GraphQL compliant client that generates Java models from sta
...
+            maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+        }
+
+        dependencies {
+            classpath dep.kotlinGradlePlugin
+        }
+    }
+    repositories {
+        jcenter()
+        maven { url "https://jitpack.io" }
```
Comment (sav007) [inline README.md]: what dependency required this maven repo?

File: README.md [hunk 3]
```diff
@@ -9,18 +9,83 @@ Apollo-Android is a GraphQL compliant client that generates Java models from sta
...
+        classpath dep.androidPlugin
+        classpath dep.apolloPlugin
+    }
 }
 
+apply plugin: 'com.android.application'
 apply plugin: 'com.apollographql.android'
+
+dependencies {
+    compile 'com.apollographql.android:api:0.1.0'
```
Comment (sav007) [inline README.md]: I believe plugin adds dep to api internally, so we don't need it explicitly, but we need to verify this

File: README.md (hunk 2 above)
Comment (digitalbuddha) [inline README.md]: removing.

**Comment by digitalbuddha** [comment]
@sav007 updated with feedback, thanks!

File: README.md [hunk 4]
```diff
@@ -9,18 +9,69 @@ Apollo-Android is a GraphQL compliant client that generates Java models from sta
...
+        classpath dep.apolloPlugin
+    }
 }
 
+apply plugin: 'com.android.application'
 apply plugin: 'com.apollographql.android'
+
+dependencies {
+    compile 'com.squareup.retrofit2:retrofit:2.1.0'
+    compile 'com.apollographql.android:converter-pojo:0.1.0'
```
Comment (sav007) [inline README.md]: compile 'com.apollographql.android:converter-pojo:0.1.0' >> compile 'com.apollographql.android:converter:0.2.1-SNAPSHOT'

Comment (marwanad) [inline README.md]: might as well use `dep. apolloConverter`

**Comment by martinbonnin** [comment]
+1, thanks :)

**Pull request merged**